apps/graph4kg/models/score_functions.py

```diff
@@ -182,10 +197,9 @@ def get_neg_score(self,
             score = emb_complex.reshape([mini_batch_num, -1, 1, hidden_dim])
             tails = tails.reshape([mini_batch_num, 1, -1, hidden_dim])
             score = score - tails
-            re_score, im_score = paddle.chunk(score, chunks=2, axis=1)
-            score = paddle.stack(
-                [re_score, im_score], axis=-1).norm(
-                    p=2, axis=-1)
+            re_score, im_score = paddle.chunk(score, chunks=2, axis=-1)
+            score = paddle.sqrt(re_score * re_score + im_score * im_score +
+                                self.epsilon)
 
             return self.gamma - score.sum(-1)
```

Comment:
The `apps/graph4kg/optimizer.py` can be deleted.

Reply:
done